src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,101 @@
+package duke;
+import java.io.IOException;
+
+public class Parser {
+    private TaskList tasks;
+    private Storage storage;
+    String dummyString;
+    Task dummyTask;
+    int counter;
+    int start;
+    int end;
```

Comment:
Should there be access modifiers for these fields? 🤔 Same with codes in other files.